### DISCUSSION TITLE
Welcome to MyPlot Discussions!

### DISCUSSION THREAD

**Discussion by jasonw4331** [jasonw4331/MyPlot#366] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Wertzui123** [comment]
Hi!

**Comment by Aericio** [comment]
Hi

**Comment by Jean-TKG** [comment]
Hi

**Comment by DHJwastaken** [comment]
i was wondering what the commands for setting the plot limit per player were bcos i cant find them anywhere and i coulda swore that was a thing b4.
also is there a way to set a price for plots for every plot world? and if so is there a way to make it stagger

**Reply by jasonw4331** [reply]
There's never been a command for it. Usually the limit is set by the permissions assigned to the player

**Reply by DHJwastaken** [reply]
can you give me an example as to how that might be done?

**Reply by jasonw4331** [reply]
https://github.com/jasonwynn10/MyPlot/wiki/Commands-And-Permissions

**Reply by DHJwastaken** [reply]
thanks a lot mate

**Reply by DHJwastaken** [reply]
> https://github.com/jasonwynn10/MyPlot/wiki/Commands-And-Permissions
apparently claimplots cant be set to a number lower than 2? i think its because it is set as a default permission in plugin.yml.
how can i change that without breaking the code?

**Reply by DHJwastaken** [reply]
im trying to set it to 1

**Reply by jasonw4331** [reply]
you need to remove the default permission before you can set any others. Check with your permission management plugin for how to do that.

**Reply by DHJwastaken** [reply]
im using pureperms and i tried to do -myplot.claimplots.2 then adding myplot.claimplots.1 but to little success.
what do you suggest might be the problem?

**Reply by DHJwastaken** [reply]
removing the perm using a perm manager still doesnt work but i managed to change the plugin.yml without breaking the plugn.
thanks anyways

**Reply by DHJwastaken** [reply]
i'd appreciate it if you set that perm to be 1 instead of 2 next time as well, it just makes more sense to be that way

**Comment by DHJwastaken** [comment]
ive noticed an issue with plot worlds when pureperms multi world perms are enabled.
if for example, a player has the perm myplot.claimplots.1 and they have a plot in a claimed world, the plugin will not allow them to claim plots else where.

**Comment by AngelliaX** [comment]
can you unban me on discordf

**Comment by KatunkaHD** [comment]
Is there a way to allow normal players to make Maps in the Plot World? Or What Permission is it?
